File: src/main/java/br/com/cvc/scheduling/transfer/model/Transfer.java
package br.com.cvc.scheduling.transfer.model;

import lombok.*;

import javax.persistence.*;
import java.io.Serializable;
import java.time.LocalDate;

/**
 * Transfer Model
 *
 * @author Victor Rodrigues de Matos
 */

@Entity
@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "Transfer")
public class Transfer implements Serializable {

    private static final long serialVersionUID = 7156526077883281623L;

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "rate", nullable = false)
    private Double rate;

    @Column(name = "value", nullable = false)
    private Double value;

    @Column(name = "transfer_date", nullable = false)
    private LocalDate transferDate;

    @Column(name = "target_account", nullable = false)
    private String targetAccount;

    @Column(name = "source_account", nullable = false)
    private String sourceAccount;

    @Column(name = "scheduling_date", nullable = false)
    private LocalDate schedulingDate;

}

File: src/main/java/br/com/cvc/scheduling/transfer/model/Transfer.kt
package br.com.cvc.scheduling.transfer.model

import java.time.LocalDate
import javax.persistence.*

@Entity
@Table(name = "Transfer")
class Transfer(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,
    @Column
    var rate: Double? = null,
    @Column
    var value: Double? = null,
    @Column
    var transferDate: LocalDate? = null,
    @Column
    var targetAccount: String? = null,
    @Column
    var sourceAccount: String? = null,
    @Column
    var schedulingDate: LocalDate? = null
)